scripts/s2i-generate.sh

```diff
@@ -13,6 +13,12 @@ source "$(dirname ${BASH_SOURCE[0]})/s2i-common.sh"
 # s2i builder image name (fully qualified)
 declare -rx S2I_BUILDER_IMAGE="${S2I_BUILDER_IMAGE:-}"
 
+# takes the values in ENV_VARS and creates an array using those values
+declare -ra env_vars=(${@})
+
+# variable representing the environemtn file 
+declare -rx ENV_DIR="/s2i-env"
```

Comment:
Let's reuse the same directory than we use to generate the `Dockerfile`, please.